01-intro.Rmd

```diff
@@ -1,2 +1,79 @@
 # Introduction {#intro}
 
+## Background
+
+Version control is an important part of any project. It allows you to reactively respond to request for changes, ensure that you can easily reproduce an earlier iteration and provides an audit trail of what work has been done and the decisions that have been made. 
+
+Most people would recognise this and probably are doing some form of their own version control to ensure they are able to, albeit not immediately, respond to these situations. 
```

Comment:
able to respond, albeit not immediately, to...

Reply:
Implemented